test/python/test_sampler.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+import numpy as np
+import pytest
+from qiskit import ClassicalRegister, QuantumCircuit, QuantumRegister
+from qiskit.circuit import Parameter
+from qiskit.transpiler.passes import Unroller
+from qiskit.transpiler.passmanager import PassManager
+
+from mqt.ddsim.sampler import DDSIMBackendSampler
+
+
+@pytest.fixture()
+def sampler() -> DDSIMBackendSampler:
+    """The sampler fixture for the tests in this file."""
+    return DDSIMBackendSampler()
+
+
+@pytest.fixture()
+def circuit() -> QuantumCircuit:
+    """The circuit fixture for the tests in this file."""
+
+    theta_a = Parameter("theta_a")
+    theta_b = Parameter("theta_b")
+    qc = QuantumCircuit(name="test")
+    q = QuantumRegister(1, "q")
+    c = ClassicalRegister(1, "c")
+    qc.add_register(q)
+    qc.add_register(c)
+    qc.rx(theta_a, 0)
+    qc.rz(theta_b, 0)
+    qc.rx(theta_a, 0)
+    qc.barrier()
+    qc.measure(q, c)
```

Comment:
```suggestion
    qc = QuantumCircuit(1, name="test")
    qc.rx(theta_a, 0)
    qc.rz(theta_b, 0)
    qc.rx(theta_a, 0)
    qc.measure_all()
```
Is way shorter and should do the same thing